src/components/Header.js

```diff
@@ -1,16 +1,72 @@
+import {useState, useEffect} from "react";
+
 export default function Header() {
+  const [isMobileNavActive, setMobileNavActive] = useState(false);
+
+  const handleMobileNavToggle = () => {
+    setMobileNavActive(!isMobileNavActive);
+  };
+
+  useEffect(function mount() {
+    if (isMobileNavActive) {
+      // When mobile nav is active, body becomes fixed to avoid scrolling
+      document.body.style.position = 'fixed';  
+    } else {
+      document.body.style.position = null;
+    }
+  });
+  
   return (
     <header
-      className="header site-padding-x inner-content-max-width mxauto flex justify-between items-center flex-row pt1"
+      className="header z-overlay relative"
     >
-      <img className="Seaborne logo" src="/img/seaborne-logo.svg" alt="Seaborne logo" />
-      <a
-        aria-label="Get in touch with Seaborne"
-        href="mailto:hello@seaborne.nyc"
-        className="event-contact-us button--style-blue-border nav-button flex items-center"
+      <div className="desktop-nav-container mxauto flex justify-between items-center flex-row pt1 px1_5 py1_5">
+        <a 
+          href="/"
+          rel="noopener noreferrer"
+          aria-label="Link to homepage"
+        >
+          <img className="Seaborne logo" src="/img/seaborne-logo.svg" alt="Seaborne logo" />
+        </a>
+        
+        <button 
+          className={`nav-toggle flex flex-col md:none bg-color-transparent ${isMobileNavActive ? "mobile-nav-active" : ""}`}
+          onClick={handleMobileNavToggle}
+        >
+          <span className="transition"></span>
+          <span className="transition"></span>
+        </button>
+
+        <nav className="desktop-nav none md:flex flex-row items-center text-body-sm">
+          <a href="/about" className="color-gray">How we work</a>
+          <a href="/studies" className="color-gray">Case Studies &amp; Blog Posts</a>
```

Comment:
Clubbed the two "case studies" and "view blog" links into one after discussing with Hugh.